apis/core/lib/csvw-builder/index.ts

```diff
@@ -30,7 +31,7 @@ interface CsvwBuildingContext {
   resources: ResourceStore
 }
 
-function mappedColumn({ csvMapping, columnMapping, column }: CsvwBuildingContext & { column: CsvColumn; columnMapping: ColumnMapping }): Initializer<Csvw.Column> {
+function mappedLiteralColumn({ csvMapping, columnMapping, column }: CsvwBuildingContext & { column: CsvColumn; columnMapping: LiteralColumnMapping }): Initializer<Csvw.Column> {
```

Comment:
Maybe this file is getting big enough to split the `mappedLiteralColumn` and `mappedReferenceColumn` into their own modules?

Reply:
I really prefer long modules than modules with one function in general, but it's up to you... you're kind of the owner of the this package ;)